### ISSUE TITLE
Actually iOS8 required

### ISSUE THREAD

**Issue by hstdt** [hons82/THCalendarDatePicker#81]
code

```
NSDateComponents* comps = [[NSCalendar calendarWithIdentifier:NSCalendarIdentifierGregorian] components:NSCalendarUnitYear|NSCalendarUnitMonth fromDate:date];
```

at `THDatePickerViewController.m` required iOS8.

**Comment by hons82** [comment]
So I'll increase the minimum to iOS8... 
What exactly is the error message that you get?

**Comment by hstdt** [comment]
@hons82  No message but crashed at iOS7.

**Comment by hons82** [comment]
That will be fixed in 1.2.9
